taxonomies/ingredients.txt

```diff
@@ -22956,6 +22969,9 @@ fr:semoule blanche de blé dur
 
 <fr:semoule blanche de blé dur
 fr:semoule de blé dur blanche biologique
+fr:semolina di grano duro bianca biologica
+fr:biologische weiße Hartweizengrieß
+fr:sémola de trigo duro blanca orgánica
```

Comment:
```suggestion
it:semolina di grano duro bianca biologica
de:biologische weiße Hartweizengrieß
es:sémola de trigo duro blanca orgánica
```

Reply:
sorry, cannot apply you changes: 
"This diff has recently been updated. Refresh and try again"
Of course, refresh does not change anything :(

Reply:
for all suggestions
also if I add suggestion to batch
sorry :(